pkg/controller/volume/persistentvolume/volume_host.go

```diff
@@ -147,3 +147,7 @@ func (ctrl *PersistentVolumeController) GetSubpather() subpath.Interface {
 func (ctrl *PersistentVolumeController) GetFilteredDialOptions() *proxyutil.FilteredDialOptions {
 	return ctrl.filteredDialOptions
 }
+
+func (ctrl *PersistentVolumeController) GetPodsNeedSyncVolumes() []types.UID {
```

Comment:
You can actually put this in KubeletVolumeHost, which will avoid needing to add blank implementations to the volume controllers

Reply:
Thanks! I will move this to KubeletVolumeHost in next commit.